src/guide/quick-start.md

```diff
@@ -22,7 +22,7 @@ footer: false
 
 In this section we will introduce how to scaffold a Vue [Single Page Application](/guide/extras/ways-of-using-vue#single-page-application-spa) on your local machine. The created project will be using a build setup based on [Vite](https://vitejs.dev) and allow us to use Vue [Single-File Components](/guide/scaling-up/sfc) (SFCs).
 
-Make sure you have an up-to-date version of [Node.js](https://nodejs.org/) installed, then run the following command in your command line (without the `>` sign):
+Make sure you have an up-to-date version of [Node.js](https://nodejs.org/) installed, then run the following command in your command line with the project root folder as the current working directory (without the `>` sign):
```

Comment:
This isn't accurate, since there is no project root folder before you run the tool. The root folder is created by the tool.

Maybe reword it to "in a directory where you intend to create the project"

Reply:
sure sir, thank you for correcting me. I shall revise the changes